java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/refactoring/ui/codicon.css

```diff
@@ -0,0 +1,485 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
```

Comment:
I assume, that the ttf file is under the same license?! Please add an entry for the `codicon.ttf` to `license-info.xml`. The same is true for other font and graphics files (I see `newHTML.png`).